site/js/modules/autograder/submissions.js

```diff
@@ -32,13 +32,15 @@ function peek(course, assignment, submission = undefined) {
     });
 }
 
-function submit(course, assignment, files) {
+function submit(course, assignment, files, allowLate = false, message) {
```

Comment:
Missing default on message.
Once you start to default, go ahead and default everything after.